compiler-rt/test/sanitizer_common/TestCases/Linux/static-link.c

```diff
@@ -0,0 +1,2 @@
+// UNSUPPORTED: hwasan, ubsan
```

Comment:
I guess `hwasan, ubsan` can be with or without interceptors

hwasan on non-android linux will be with interceptors

Reply:
Right. The test is only for sanitizers that are guaranteed to use interceptors that are incompatible with `-static` links. I unsupport hwasan/ubsan because they may not use interceptors.

Reply:
I see, thanks for explaining!